docs/getting-started/cv-deploy-porting.md

```diff
@@ -147,5 +147,56 @@ We recommend leveraging the `cv_server` and `cv_token` keys to specify the authe
 
 ```
 
+## Uploading Static Configlets
+
+The `cv_deploy` role also replaces the functionality of the legacy `cvp_configlet_upload` role. Instead of discovering configlets in a directory, `cv_deploy` uses an explicit manifest to manage configlets and containers in the CloudVision **Static Configuration Studio**.
+
+To manage only the Static Configuration Studio content, you run in **"manifest-only" mode** by setting `cv_devices: []`. This instructs the role to skip all device-specific operations and only process the manifest. Note that the playbook task still requires the standard connection variables and can use other keys like `cv_run_change_control` specified in the previous section.
```

Comment:
This is quite confusing to me that we only mention `manifest-only` mode and that it comes first.

I would add a couple of things here:

1. explain that there are two modes in one or two sentences and what is the difference. 
2. states clearly that to replace `cvp_configlet_upload` task you _need_ to run in `manifest-only` mode.
3. What do I need to target as devices in my playbook  when running in manifest-only mode. It looks like you are only putting one task in old and new. Do I need to set hosts to `FABRIC`, does it even matter in that case? (if not we should say so :) )

Reply:
> states clearly that to replace cvp_configlet_upload task you need to run in manifest-only mode.

Well that depends. If you are using cvp_configlet_upload in combination with eos_config_deploy_cvp it would not be manifest-only.

Reply:
the only example below is manifest-only so I would be confused reading this as to what I should do